meshnets/process.py

```diff
@@ -75,9 +75,30 @@ def main(_):
             folder, FLAGS.interpolated_pressure_field_name)
         interpolated_mesh.save(interpolated_mesh_path)
 
-        graph_data = data_processing.mesh_file_to_graph_data(
-            interpolated_mesh_path, WIND_VECTOR, load_pressure=True)
+        edge_index, edge_attr = data_processing.make_edge_index_and_features(
+            interpolated_mesh_path)
+
+        wind_vector_path = os.path.join(folder, FLAGS.wind_vector_name)
+        if os.path.exists(wind_vector_path):
+            with open(wind_vector_path, encoding='utf-8') as f:
+                wind_vector = json.load(f)
+        else:
+            wind_vector = FLAGS.default_wind_vector
+        node_features = data_processing.make_node_features(
+            interpolated_mesh_path, wind_vector)
 
+        target = data_processing.make_target(interpolated_mesh_path)
+
+        # Save as numpy arrays.
+        np.save(os.path.join(folder, 'edge_index.npy'), edge_index)
+        np.save(os.path.join(folder, 'edge_attr.npy'), edge_attr)
+        np.save(os.path.join(folder, 'node_features.npy'), node_features)
+        np.save(os.path.join(folder, 'target.npy'), target)
```

Comment:
I think I wouldn't use such generic names, and probably it would not even hardcode in the name that one of them is the prediction target (unless that's a requirement?), because I can see the dataset being used in different ways. For example, when you add here the drag coefficient, or stream lines, perhaps those will be the targets rather than the pressure.
So, I would use names that are more descriptive of the contents of the data: such as pressure....

Reply:
I will address the hard coded names. Regarding the names themselves I agree that target should be renamed to something like `wind_pressure`. The others seem to me to be pretty standard